mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -834,6 +818,34 @@ def test_filter():
     data, _ = raw[picks, :]
     assert_array_almost_equal(data, data_notch, sig_dec_notch_fit)
 
+    # filter should set the "lowpass" and "highpass" parameters
+    raw = RawArray(np.random.randn(3, 1000),
+                   create_info(3, 1000., ['eeg'] * 2 + ['stim']))
+    raw.info['lowpass'] = raw.info['highpass'] = None
+    for kind in ('lowpass', 'highpass'):
+        if kind == 'lowpass':
+            l_freq, h_freq = None, 50
+        else:
+            l_freq, h_freq = 50, None
```

Comment:
I like this better:
`        l_freq, h_freq = (None, 50) if kind == 'lowpass' (else 50, None)`